src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  let discount = 0;
+
+  switch (true) {
+    case days >= 3 && days < 7:
+      discount = 20;
+      break;
+    case days >= 7:
+      discount = 50;
+      break;
+  }
```

Comment:
Replace the `switch` statement with `if` statements to simplify the logic. Also, avoid using hardcoded values for the number of days and the discount amounts. Define them as constants at the top of your function to make the code more maintainable and self-explanatory.